lib/membrane_ffmpeg_swresample_plugin/converter.ex

```diff
@@ -9,54 +9,56 @@ defmodule Membrane.FFmpeg.SWResample.Converter do
 
   require Membrane.Logger
 
-  alias Membrane.{Buffer, RawAudio, RemoteStream}
-  alias Membrane.Caps.Matcher
   alias __MODULE__.Native
+  alias Membrane.{Buffer, RawAudio, RemoteStream}
 
-  @supported_out_caps {RawAudio,
-                       sample_format: Matcher.one_of([:u8, :s16le, :s32le, :f32le, :f64le]),
-                       channels: Matcher.one_of([1, 2])}
+  @supported_format [:u8, :s16le, :s32le, :f32le, :f64le]
```

Comment:
It might be confusing to get the difference between `supported_format` and `accepted_format` so maybe let's rename to `supported_sample_formats` to also be consistent with `supported_channels` (plural)

```suggestion
  @supported_sample_format [:u8, :s16le, :s32le, :f32le, :f64le]
```